<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Livertool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget0">
      <widget class="QWidget" name="tabWidget0_Page1">
       <attribute name="title">
        <string>File name</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="ImageView" name="pcv_img"/>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Navigation</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Slice</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSlider" name="sld_slice">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="spb_slice"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Echo</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSlider" name="sld_echo">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="spb_echo"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="information_tab">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radio_summary_header">
            <property name="text">
             <string>Summary</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radio_full_header">
            <property name="text">
             <string>View header file</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTextEdit" name="textedit_header">
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fatmap_tab">
       <attribute name="title">
        <string>Fat map</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="ImageView" name="pcv_fatmap">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Settings</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0" alignment="Qt::AlignLeft">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Fat Fraction estimation method:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cbx_methods"/>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="check_current_slice">
               <property name="toolTip">
                <string/>
               </property>
               <property name="toolTipDuration">
                <number>-1</number>
               </property>
               <property name="whatsThis">
                <string/>
               </property>
               <property name="text">
                <string>Map only current slice</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="btn_cfatmap">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>check_current_slice</zorder>
            <zorder>label_4</zorder>
            <zorder>cbx_methods</zorder>
            <zorder>btn_cfatmap</zorder>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="signal_tab">
       <attribute name="title">
        <string>ROI</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_roi_delete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Data and Fat fraction estimation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QComboBox" name="cbox_rois"/>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="PlotWidget" name="pcv_signal">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Signal Intensity (a.u.) vs Echo time (ms)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QTextEdit" name="textEdit_roi"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="export_tab">
       <attribute name="title">
        <string>Export results</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>General</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="check_anonimyze">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Removes the name of the patient of the header and summary files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Anonimyze data</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exports separate DICOM files for each dataset and estimation method, or separate bitmap images for each mapped slice in each method.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="title">
             <string>Fat fraction maps</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <widget class="QRadioButton" name="radio_export_dicom">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exporting in DICOM is available only for results performed over the whole dataset. The file will inherit the original header with few modifications.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="whatsThis">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>DICOM</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="radio_export_bitmap">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exporting in bitmap is available for when only selected slices of the dataset were mapped using one of the algorithms.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Bitmap</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="check_export_dicom">
               <property name="text">
                <string>Dixon</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="check_export_triple">
               <property name="text">
                <string>Triple</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="check_export_multi">
               <property name="text">
                <string>Multi</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="check_export_ann">
               <property name="text">
                <string>ANN</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>ROI information</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_13">
             <item row="0" column="0">
              <widget class="QCheckBox" name="check_export_csv">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exports a csv (comma separated values) file containing information computed about the selected regions of interest, if any. Includes summary information about the test.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Export .CSV</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="check_include_csv_hdr">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exports a csv (comma separated values) file containing information computed about the selected regions of interest, if any. Includes summary information about the test.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Include columns header</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="btn_export">
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLivertool">
    <property name="title">
     <string>File...</string>
    </property>
    <addaction name="act_open_single"/>
    <addaction name="act_open_multiple"/>
    <addaction name="separator"/>
    <addaction name="act_open_parrec"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuLivertool"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_rectangle"/>
   <addaction name="act_ellipse"/>
   <addaction name="act_polygon"/>
   <addaction name="act_ROI_done"/>
   <addaction name="act_ROI_cancel"/>
   <addaction name="separator"/>
  </widget>
  <action name="act_open_single">
   <property name="text">
    <string>Open a single DICOM file...</string>
   </property>
  </action>
  <action name="act_rectangle">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-Rectangular.png</normaloff>icons/icons8-Rectangular.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="act_ellipse">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-Oval Filled.png</normaloff>icons/icons8-Oval Filled.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="act_polygon">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-Polygon Filled.png</normaloff>icons/icons8-Polygon Filled.png</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
  </action>
  <action name="act_ROI_done">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-Ok.png</normaloff>icons/icons8-Ok.png</iconset>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </action>
  <action name="act_ROI_cancel">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-Cancel_2.png</normaloff>icons/icons8-Cancel_2.png</iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </action>
  <action name="act_open_multiple">
   <property name="text">
    <string>Open multiple DICOM files...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export results...</string>
   </property>
  </action>
  <action name="act_open_parrec">
   <property name="text">
    <string>Open PAR /REC files...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sld_slice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_slice</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_slice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_slice</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_echo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_echo</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_echo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_echo</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
